### DISCUSSION TITLE
Bugs and Glitches Fixed?

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2705] [General]
Hello, I used Explorer Patcher about a month and a half ago, and when I downloaded it, used it for a couple hours, the program corrupted something in my File Explorer, which made my files on the left side of File Explorer disappear. I later on fixed it, but I decided to uninstall Explorer Patcher because of this, so this is why I am asking if bugs were fixed or not, I am pretty sure this is happening because of Microsoft updating some things to the taskbar.

**Comment by pyrates999** [comment]
Install EP and find out.  Be sure you download the newest release of EP first.

Please post windows version and build.  You can find this by running winver.

Please include any other software you have installed that modifies the UI of windows.
